cmd/nvidia-dra-controller/imex.go

```diff
@@ -312,3 +310,57 @@ func (m *ImexManager) cleanupResourceSlices() error {
 
 	return nil
 }
+
+// add sets the offset where an IMEX domain's channels should start counting from.
+func (offsets imexDomainOffsets) add(imexDomain string, resourceSliceImexChannelLimit, driverImexChannelLimit int) (int, error) {
+	id := strings.SplitN(imexDomain, ".", 2)
+	if len(id) != 2 {
+		return -1, fmt.Errorf("error adding IMEX domain %s: invalid format", imexDomain)
+	}
+	imexDomain = id[0]
+	cliqueID := id[1]
+
+	// Check if the IMEX domain is already in the map.
```

Comment:
```suggestion
	// Check if the IMEX domain is already in the map
```

Reply:
use of habit, golangci-lint complains about the dot, but I guess is just for the func level comment